settings.gradle.kts

```diff
@@ -156,7 +154,7 @@ pluginManagement {
                     useModule("com.android.tools.build:gradle:3.5.3")
 
                 pluginId.startsWith("org.jetbrains.kotlin.") ->
-                    useVersion(kotlinVersion)
+                    useVersion("1.3.70")
```

Comment:
Do we have access to Dependencies.kt?

Reply:
no afaik

Reply:
https://docs.gradle.org/current/userguide/upgrading_version_5.html#classes_from_buildsrc_are_no_longer_visible_to_settings_scripts